src/Essentials/src/Geolocation/Geolocation.tizen.cs

```diff
@@ -67,5 +69,11 @@ public async Task<Location> GetLocationAsync(GeolocationRequest request, Cancell
 
 			return lastKnownLocation;
 		}
+
+		public Task<bool> StartListeningForegroundAsync(GeolocationListeningRequest request) =>
+			throw ExceptionUtils.NotSupportedOrImplementedException;
+
+		public void StopListeningForeground() =>
+			throw ExceptionUtils.NotSupportedOrImplementedException;
```

Comment:
@JoonghyunCho can you or someone on your team look at this maybe?

Reply:
Oops, I almost missed the comment. Thanks for letting me know!
Currently, there is no actual device target I can run maui and use the real time location service.
Let me leave it as is for now, and see if I can implement with any emulator.
Thanks!